tests/test_core.py

```diff
@@ -158,13 +158,13 @@ def test_temporal_datatypes():
         metadata_obj.create_all(conn)
         insert = table.insert().values(
             column_date="2022-03-19",
-            column_time="06:04:19.222",
```

Comment:
We do want to keep the time here as we've had issues with this before. Is there a reason this was changed?

Reply:
MySQL `TIME` datatype doesn't support fractional seconds by default, so I removed the fractional component. I've changed the test to use `TIME(timezone=False, fsp=6)` so that fractional seconds can be tested for.